### PR TITLE
[HTML5 2.0] User-list Redesign

### PR DESCRIPTION
**Pull request by oswaldoacauan**
Implement new design for the user-list/user-avatar.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx [hunk 1]
```diff
@@ -288,41 +245,27 @@ class UserListItem extends Component {
...
         </span>
         <span className={styles.userNameSub}>
-          {userNameSub}
-          {(user.isLocked) ?
-            <span> {(user.isCurrent ? ' | ' : null)}
-              <Icon iconName="lock" />
-              {intl.formatMessage(messages.locked)}
-            </span> : null}
+          {userNameSub
+            .reduce((prev, curr) => [prev, ' | ', curr])}
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx]: This line is throwing me errors, whatever i try open the userlist

screenshot from 2017-07-31 16-57-44

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx [hunk 2]
```diff
@@ -263,10 +263,13 @@ class UserListItem extends Component {
         <span className={styles.userNameMain}>
           {user.name} <i>{(user.isCurrent) ? `(${intl.formatMessage(messages.you)})` : ''}</i>
         </span>
-        <span className={styles.userNameSub}>
-          {userNameSub
-            .reduce((prev, curr) => [prev, ' | ', curr])}
-        </span>
+        {
+          userNameSub ?
```
Comment (Klauswk) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx]: Empty array are not falsy objects, you must check for its length.

**Comment by Klauswk** [comment]
When i close the chat, the `<` remains, and if pressed, breaks the user list

screenshot from 2017-08-02 14-17-13

screenshot from 2017-08-02 14-17-26

**Comment by oswaldoacauan** [comment]
@Klauswk I actually added that button just to test the compact view commented it out on 92da0ac

**Pull request merged**